src/apps/competitions/tests/test_phase_migration.py

```diff
@@ -95,3 +121,48 @@ def test_all_submissions_migrated_before_changing_phase_status(self):
 
         mock_start = self.mock_migration()
         assert mock_start.call_count == 0
+
+
+class PhaseStatusTests(TestCase):
+    def setUp(self):
+        self.user = UserFactory()
+        self.comp = CompetitionFactory(created_by=self.user)
+        self.tasks = [TaskFactory(created_by=self.user)]
+        base = {'competition': self.comp, 'tasks': self.tasks}
+        self.phase1 = PhaseFactory(
+            **base,
+            start=now() - timedelta(minutes=10),
+            end=now() - timedelta(minutes=5),
+        )
+        self.phase2 = PhaseFactory(
+            **base,
+            start=now() - timedelta(minutes=1),
+            end=now() + timedelta(minutes=5),
+        )
+        self.phase3 = PhaseFactory(
+            **base,
+            start=now() + timedelta(minutes=10),
+            end=now() + timedelta(minutes=15),
+        )
+
+    def test_phase_migration_updates_statuses(self):
```

Comment:
Should probably have a test where phase1 status is `Phase.CURRENT`, phase2 status is `Phase.NEXT` and phase3 status is checked. 

Also might be worth checking statuses on competitions where there are 5 or more phases?